source/llvm/LLVMExecutableModel.cpp

```diff
@@ -2239,6 +2284,13 @@ double LLVMExecutableModel::getRandom()
     return (*modelData->random)();
 }
 
+void LLVMExecutableModel::saveState(std::ostream& out)
```

Comment:
Should this function be hidden from users? i.e. modify the SWIG interface file to ignore it.